Batteries/Data/BitVec/Basic.lean

```diff
@@ -0,0 +1,19 @@
+/-
+Copyright (c) 2024 François G. Dorais. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: François G. Dorais
+-/
+
+import Batteries.Data.Fin.OfBits
+
+namespace BitVec
+
+/-- `ofFnLEAux f` returns the `BitVec m` whose `i`th bit is `f i` when `i < m`, little endian. -/
+@[inline] def ofFnLEAux (m : Nat) (f : Fin n → Bool) : BitVec m :=
```

Comment:
I'm not quite sure whether we want to have these functions in general. There is no chance that `bv_decide` can ever support this kind of `BitVec` as it would require reasoning about a function `Fin n -> Bool` which it cannot do. Whether this is an inclusion/exclusion criterium for `BitVec` stuff I don't know.

Reply:
I'm using these for linear algebra over the 2-element field, so this is not completely related to `bv_decide`.